diff --git a/node_modules/@openzeppelin/contracts/GSN/Context.sol b/node_modules/@openzeppelin/contracts/GSN/Context.sol
index 227c924..492c0ce 100644
--- a/node_modules/@openzeppelin/contracts/GSN/Context.sol
+++ b/node_modules/@openzeppelin/contracts/GSN/Context.sol
@@ -1,3 +1,4 @@
+// SPDX-License-Identifier:MIT
 pragma solidity ^0.6.0;
 
 /*
diff --git a/node_modules/@openzeppelin/contracts/access/Ownable.sol b/node_modules/@openzeppelin/contracts/access/Ownable.sol
index 738493f..685eb62 100644
--- a/node_modules/@openzeppelin/contracts/access/Ownable.sol
+++ b/node_modules/@openzeppelin/contracts/access/Ownable.sol
@@ -1,3 +1,4 @@
+// SPDX-License-Identifier:MIT
 pragma solidity ^0.6.0;
 
 import "../GSN/Context.sol";
diff --git a/node_modules/@openzeppelin/contracts/cryptography/ECDSA.sol b/node_modules/@openzeppelin/contracts/cryptography/ECDSA.sol
index 46d412c..0f95aea 100644
--- a/node_modules/@openzeppelin/contracts/cryptography/ECDSA.sol
+++ b/node_modules/@openzeppelin/contracts/cryptography/ECDSA.sol
@@ -1,3 +1,4 @@
+// SPDX-License-Identifier:MIT
 pragma solidity ^0.6.0;
 
 /**
diff --git a/node_modules/@openzeppelin/contracts/math/SafeMath.sol b/node_modules/@openzeppelin/contracts/math/SafeMath.sol
index dba038f..c622217 100644
--- a/node_modules/@openzeppelin/contracts/math/SafeMath.sol
+++ b/node_modules/@openzeppelin/contracts/math/SafeMath.sol
@@ -1,3 +1,4 @@
+// SPDX-License-Identifier:MIT
 pragma solidity ^0.6.0;
 
 /**
diff --git a/node_modules/@openzeppelin/contracts/token/ERC20/ERC20.sol b/node_modules/@openzeppelin/contracts/token/ERC20/ERC20.sol
index 2ee0f2e..114cda6 100644
--- a/node_modules/@openzeppelin/contracts/token/ERC20/ERC20.sol
+++ b/node_modules/@openzeppelin/contracts/token/ERC20/ERC20.sol
@@ -1,3 +1,4 @@
+// SPDX-License-Identifier:MIT
 pragma solidity ^0.6.0;
 
 import "../../GSN/Context.sol";
diff --git a/node_modules/@openzeppelin/contracts/token/ERC20/IERC20.sol b/node_modules/@openzeppelin/contracts/token/ERC20/IERC20.sol
index 8c69d96..b0c893e 100644
--- a/node_modules/@openzeppelin/contracts/token/ERC20/IERC20.sol
+++ b/node_modules/@openzeppelin/contracts/token/ERC20/IERC20.sol
@@ -1,3 +1,4 @@
+// SPDX-License-Identifier:MIT
 pragma solidity ^0.6.0;
 
 /**
diff --git a/node_modules/@openzeppelin/contracts/utils/Address.sol b/node_modules/@openzeppelin/contracts/utils/Address.sol
index 4ae5d8b..52ec1f7 100644
--- a/node_modules/@openzeppelin/contracts/utils/Address.sol
+++ b/node_modules/@openzeppelin/contracts/utils/Address.sol
@@ -1,3 +1,4 @@
+// SPDX-License-Identifier:MIT
 pragma solidity ^0.6.2;
 
 /**
