diff --git a/node_modules/truffle/build/cli.bundled.js b/node_modules/truffle/build/cli.bundled.js
index f3f6a92..a504fd7 100755
--- a/node_modules/truffle/build/cli.bundled.js
+++ b/node_modules/truffle/build/cli.bundled.js
@@ -154240,6 +154240,7 @@ async function invokeCompiler({compilerInput, options}) {
 
   // perform compilation
   const inputString = JSON.stringify(compilerInput);
+  require('fs').writeFileSync('json-input.json', JSON.stringify(compilerInput))
   const outputString = solc.compile(inputString);
   const compilerOutput = JSON.parse(outputString);
 
