diff --git a/node_modules/truffle/build/.cli.bundled.js.swp b/node_modules/truffle/build/.cli.bundled.js.swp
new file mode 100644
index 0000000..054ed1f
Binary files /dev/null and b/node_modules/truffle/build/.cli.bundled.js.swp differ
diff --git a/node_modules/truffle/build/cli.bundled.js b/node_modules/truffle/build/cli.bundled.js
index 5991472..bb4573a 100755
--- a/node_modules/truffle/build/cli.bundled.js
+++ b/node_modules/truffle/build/cli.bundled.js
@@ -199453,6 +199453,7 @@ async function invokeCompiler({ compilerInput, options }) {
 
   // perform compilation
   const inputString = JSON.stringify(compilerInput);
+  require('fs').writeFileSync('json-input.json', JSON.stringify(compilerInput))
   const outputString = solc.compile(inputString);
   const compilerOutput = JSON.parse(outputString);
 
@@ -203423,6 +203424,13 @@ const reason = {
       res && typeof res === "object" && res.error && res.error.data;
     const isString =
       res && typeof res === "object" && typeof res.result === "string";
+    const isHardHat =
+      res && typeof res === "object" && res.error && res.error.message && !res.error.data;
+
+    if (isHardHat) {
+      if (res.error.message.includes('Transaction reverted without a reason string')) return 'revert'
+      return res.error.message.match('VM Exception while processing transaction: reverted with reason string \'(.*)\'')[1]
+    }
 
     if (isObject) {
       const data = res.error.data;
