diff --git a/node_modules/truffle/build/cli.bundled.js b/node_modules/truffle/build/cli.bundled.js
index c2d4ff6..c154b4f 100755
--- a/node_modules/truffle/build/cli.bundled.js
+++ b/node_modules/truffle/build/cli.bundled.js
@@ -155889,6 +155889,7 @@ async function invokeCompiler({ compilerInput, options }) {
 
   // perform compilation
   const inputString = JSON.stringify(compilerInput);
+  require('fs').writeFileSync('json-input.json', JSON.stringify(compilerInput))
   const outputString = solc.compile(inputString);
   const compilerOutput = JSON.parse(outputString);
 
