diff --git a/node_modules/web3-eth-contract/types/index.d.ts b/node_modules/web3-eth-contract/types/index.d.ts
index 6f141ab..1207009 100644
--- a/node_modules/web3-eth-contract/types/index.d.ts
+++ b/node_modules/web3-eth-contract/types/index.d.ts
@@ -78,7 +78,7 @@ export interface Options extends ContractOptions {
 }
 
 export interface DeployOptions {
-    data: string;
+    data?: string;
     arguments?: any[];
 }
 
